library/kafka_stat_lag.py

```diff
@@ -0,0 +1,251 @@
+
+# Copyright: (c) 2020, Yassine MILHI
+# GNU General Public License v3.0+
+# (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+"""
+Ansible module for consumer group statistics
+"""
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+# import module snippets
+import os
+import json
+from pkg_resources import parse_version
+
+from ansible.module_utils.basic import AnsibleModule
+from ansible.module_utils.pycompat24 import get_exception
+
+from ansible.module_utils.kafka_consumer_lag import KafkaConsumerLag
+from ansible.module_utils.kafka_manager import KafkaManager
+from ansible.module_utils.ssl_utils import generate_ssl_object
+from ansible.module_utils.ssl_utils import generate_ssl_context
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.0'}
+
+DOCUMENTATION = '''
+---
+module: kafka_stat_lag
+short_description: Gather kafka statistics
+description:
+     - Gather kafka statistics.
+     - Not compatible with Kafka version < 0.11.0.
+author:
+    - Yassine MILHI
+options:
+  consummer_group:
+    description:
+      - 'one consumer group name.'
+    required: True
+  bootstrap_servers:
+    description:
+      - 'kafka broker connection.'
+      - 'format: host1:port,host2:port'
+    required: True
+  api_version:
+    description:
+      - 'kafka version'
+      - 'format: major.minor.patch. Examples: 0.11.0 or 1.0.1'
+      - 'if not set, will launch an automatic version discovery but can '
+      - 'trigger stackstraces on Kafka server.'
+    default: auto
+  sasl_mechanism:
+    description:
+        - 'when using sasl, whether use PLAIN or GSSAPI.'
+    default: PLAIN
+    choices: [PLAIN, GSSAPI]
+  security_protocol:
+    description:
+      - 'how to connect to Kafka.'
+     default: PLAINTEXT
+     choices: [PLAINTEXT, SASL_PLAINTEXT, SSL, SASL_SSL]
+  sasl_plain_username:
+    description:
+      - 'when using security_protocol = ssl, username to use.'
+  sasl_plain_password:
+    description:
+      - 'when using security_protocol = ssl, password for '
+      - 'sasl_plain_username.'
+  ssl_check_hostname:
+    description:
+      - 'when using ssl for Kafka, check if certificate for hostname is '
+      - 'correct.'
+    default: True
+  ssl_cafile:
+    description:
+      - 'when using ssl for Kafka, content of ca cert file or path to ca '
+      - 'cert file.'
+  sasl_kerberos_service_name:
+    description:
+      - 'when using kerberos, service name.'
+  ssl_certfile:
+    description:
+      - 'when using ssl for Kafka, content of cert file or path to server '
+      - 'cert file.'
+  ssl_keyfile:
+    description:
+      - 'when using ssl for kafka, content of keyfile or path to server '
+      - 'cert key file.'
+  ssl_password:
+    description:
+      - 'when using ssl for Kafka, password for ssl_keyfile.'
+  ssl_crlfile:
+    description:
+      - 'when using ssl for Kafka, content of crl file or path to cert '
+      - 'crl file.'
+'''
+
+EXAMPLES = '''
```

Comment:
Can you also add this example and information about the new module in the main README.md file?

Reply:
Done